code/game/objects/random/rig.dm

```diff
@@ -90,10 +89,6 @@
 	/obj/item/rig_module/fabricator/energy_net = 0.2,
 	/obj/item/rig_module/self_destruct = 1,
 
-	//Stealth
-	/obj/item/rig_module/stealth_field = 0.5,
-	/obj/item/rig_module/teleporter = 0.5,
```

Comment:
I think stealth field could stay. It's an awful power hog, making it useless for anything besides just "getting a drop" on someone - and I think that having a piece of rare loot enable that is fine.

Reply:
There's a second spawner called "rare rig module" which still spawns these.

Reply:
Will resolve.